DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/Utils/UIImage+SVG.swift

```diff
@@ -0,0 +1,178 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import UIKit
+
+// NB: This extension is tailored to our specific use case, SVG
+//     containing a single shape with a fill color.
+//     For other use cases or full SVG support we should consider a
+//     3rd party dependency.
+
+extension UIImage {
+    convenience init?(svgData: Data, scale: CGFloat) {
+        guard let document = SVGShapeDocument(svgData: svgData) else {
+            return nil
+        }
+
+        let format = UIGraphicsImageRendererFormat()
+        format.scale = scale
+        format.opaque = false
+
+        let renderer = UIGraphicsImageRenderer(size: document.size, format: format)
+        let image = renderer.image { _ in
+            let path = UIBezierPath(cgPath: document.path)
+            path.usesEvenOddFillRule = document.isEOFilled
+
+            document.fillColor.setFill()
+            path.fill()
+        }
+
+        self.init(cgImage: image.cgImage!, scale: image.scale, orientation: .up)
```

Comment:
probably worth guarding cgImage instead of force unwrap

Reply:
Done!